start

```diff
@@ -377,31 +375,49 @@ function init_horizon() {
   fi
   pushd $HZHOME
 
-  run_silent "chown-horizon" chown stellar:stellar .
-
   sed -ri \
     -e "s/__PGPASS__/$PGPASS/g" \
     -e "s/__NETWORK__/$NETWORK_PASSPHRASE/g" \
     -e "s=__ARCHIVE__=$HISTORY_ARCHIVE_URLS=g" \
     etc/horizon.env
 
   if [ "$ENABLE_HORIZON_CAPTIVE_CORE" = "true" ]; then
+    
+    HORIZON_CAPTIVE_CORE_CFG=$COREHOME/etc/horizon-captive-core.cfg
+    cp $COREHOME/etc/stellar-captive-core.cfg $HORIZON_CAPTIVE_CORE_CFG 2>/dev/null || true
```

Comment:
❗ I think the cp and modify commands highlight that we've got the .cfg file in the wrong place. Instead of having a shared stellar-captive-core.cfg in the core/etc directory, which is not actually used by the stellar-core node, we should have the cfg in the horizon and soroban-rpc config directories, and then we can configure them independently without having to have a general file that get modified. It'll keep the start script simpler and the process of what's happening less complicated.

If we do this, it will also mean we won't need to modify the existing Horizon captive core config so much as part of this change.

Reply:
yes, put a copy of captive core .cfg file in specific network/service permutations that allow or use captive cores - https://github.com/stellar/quickstart/pull/414/commits/f6e66c3d67b23383a036ce3db183b4b1f9ef910c